nixos/modules/services/web-apps/code-server.nix

```diff
@@ -0,0 +1,109 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.code-server;
+
+in {
+  ###### interface
+  options = {
+    services.code-server = {
+      enable = mkEnableOption "enable code-server";
+
+      package = mkOption {
+        default = pkgs.code-server;
+        defaultText = "pkgs.code-server";
+        description = "Which code-server derivation to use";
+        type = types.package;
+      };
+
+      packages = mkOption {
+        default = [ ];
+        defaultText = "[]";
```

Comment:
I don't think you need that here... 🤔

Reply:
Well, its really neede for usage of code-server.
Problem is, that code-server is started by systemd. Lets say we would like to develop in go ( i do this in my day to day job ) problem is, that code-server is started by systemd. In the env of systemd there is no PATH to the `go` binary, even if its installed as system-package in nixos. Thats why i use the `path` option in the systemd-service. 
I was thinking if i can use `environment.systemPackages` for that :thinking: